From 3e7bb855d14330f2d56b366c16f49f769b250d63 Mon Sep 17 00:00:00 2001
From: ohaiibuzzle <23693150+ohaiibuzzle@users.noreply.github.com>
Date: Thu, 15 Jun 2023 15:14:04 +0700
Subject: [PATCH] feat: enable sdl2

---
 Formula/game-porting-toolkit.rb | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Formula/game-porting-toolkit.rb b/Formula/game-porting-toolkit.rb
index 94fc1c4..0f89859 100644
--- a/Formula/game-porting-toolkit.rb
+++ b/Formula/game-porting-toolkit.rb
@@ -95,6 +95,7 @@ class GamePortingToolkit < Formula
                               "--disable-win16",
                               "--disable-tests",
                               "--without-x",
+                              "--with-sdl",
                               "--without-pulse",
                               "--without-dbus",
                               "--without-inotify",
-- 
2.39.2 (Apple Git-144)

