--- a/Formula/game-porting-toolkit.rb
+++ b/Formula/game-porting-toolkit.rb
@@ -83,7 +83,8 @@ class GamePortingToolkit < Formula
     ENV.append "GSTREAMER_LIBS", "-lglib-2.0 -lgmodule-2.0 -lgstreamer-1.0 -lgstaudio-1.0 -lgstvideo-1.0 -lgstgl-1.0 -lgobject-2.0"
 
-    # We also need to tell the linker to add Homebrew to the rpath stack.
+    # Set rpath for Whisky
-    ENV.append "LDFLAGS", "-lSystem -L#{HOMEBREW_PREFIX}/lib -Wl,-rpath,#{HOMEBREW_PREFIX}/lib -Wl,-rpath,@executable_path/../lib/external"
+    ENV.append "LDFLAGS", "@loader_path/../lib"
 
     # Common compiler flags for both Mach-O and PE binaries.
     ENV.append_to_cflags "-O3 -Wno-implicit-function-declaration -Wno-format -Wno-deprecated-declarations -Wno-incompatible-pointer-types"