--- a/Formula/game-porting-toolkit.rb
+++ b/Formula/game-porting-toolkit.rb
@@ -55,7 +55,8 @@ class GamePortingToolkit < Formula
                   "libusb",
                   "gettext",
                   "openssl@1.1",
-                  "sane-backends"]
+                  "sane-backends",
+                  "molten-vk"]
   @@named_deps.each do |dep|
     depends_on dep
   end
@@ -95,6 +96,8 @@ class GamePortingToolkit < Formula
                               "--disable-win16",
                               "--disable-tests",
                               "--without-x",
+                              "--with-sdl",
+                              "--with-vulkan",
                               "--without-pulse",
                               "--without-dbus",
                               "--without-inotify",
@@ -116,9 +119,6 @@ class GamePortingToolkit < Formula
                                 "--without-sane",
                                 "--without-krb5",
                                 "--disable-winedbg",
-                                "--without-vulkan",
-                                "--disable-vulkan_1",
-                                "--disable-winevulkan",
                                 "--without-openal",
                                 "--without-unwind",
                                 "--without-usb"]